### PR TITLE
LPS-63229 Improve security transforming DataProviderInstances JSON return

### PR DESCRIPTION
**Pull request by mclomello**
@natocesarrego could u review this pr?

### CODE REVIEW

**Comment by mclomello** [comment]
ci:test:sf

**Comment by mclomello** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/service/impl/DDMDataProviderInstanceServiceImpl.java [hunk 1]
```diff
@@ -196,6 +202,52 @@ public DDMDataProviderInstance updateDataProviderInstance(
...
+
+		if (JSONFactoryUtil.getJSONFactory() == null) {
+			new JSONFactoryUtil().setJSONFactory(new JSONFactoryImpl());
+		}
+
+		for (DDMDataProviderInstance ddmDataProviderInstance :
+				ddmDataProviderInstances) {
+
+			try {
+				JSONObject definitionJSONObject =
```
Comment (jeyvison) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/service/impl/DDMDataProviderInstanceServiceImpl.java]: Check if we can replace all this logic by 

`JSONUtil.replace(...)`

**Comment by jeyvison** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**